psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -452,6 +452,10 @@ def scf_iterate(self, e_conv=None, d_conv=None):
             self.Da().print_out()
             self.Db().print_out()
 
+        # Set the current energy and density convergence values
+        core.set_variable("SCF DELTA E", Ediff)
```

Comment:
Why add these two variables? They didn't exist in the code before this PR, and they aren't used elsewhere in this PR.

Reply:
Ah, the DELTA D is used to determine when we can turn incremental fock build off, when we get close to convergence. Since I added DELTA D, I thought that it would make sense to add DELTA E as well. Personally, I do not like having to go to python to get delta D, and if you have any workarounds for this, I would love to hear it. :)

Reply:
Second removing `SCF DELTA E`. If we can't find another way to pass the rms density other than by global qcvar, I think it'd be better to `core.del_variable("SCF RMS D")` when it's no longer needed (after checking `has_variable`, I think).